src/pages/my_page/user_profile/ResearchProfilePanel.tsx

```diff
@@ -21,14 +20,12 @@ interface ResearchProfilePanelProps {
 
 export const ResearchProfilePanel = ({ person, isLoadingPerson }: ResearchProfilePanelProps) => {
   const { t } = useTranslation();
-  const location = useLocation();
 
   const fullName = person?.names ? getFullCristinName(person.names) : '';
   const orcidUri = getOrcidUri(person?.identifiers);
   const activeAffiliations = person?.affiliations ? filterActiveAffiliations(person.affiliations) : [];
-
-  const currentPath = location.pathname.replace(/\/$/, '').toLowerCase();
-  const isPreview = currentPath === UrlPathTemplate.MyPageMyPersonalia;
+  const personBackground = getLanguageString(person?.background) ?? '';
```

Comment:
denne skal alltid returnere en streng mener jeg, så vi ikke trenger en fallback(?)
```suggestion
  const personBackground = getLanguageString(person?.background);
```